crates/daphne-server/src/storage_proxy_connection/kv/mod.rs

```diff
@@ -212,12 +212,8 @@ impl<'h> Kv<'h> {
         let key = Self::to_key::<P>(key);
         tracing::debug!(key, "GET");
         match self.cache.read().await.get::<P>(&key) {
-            cache::GetResult::NotFound { read_through } => {
-                if !read_through {
-                    return Ok(None);
-                }
-            }
-            cache::GetResult::Found(t) => return Ok(Some(mapper(t))),
+            cache::GetResult::NotFound => {}
+            cache::GetResult::Found(t) => return Ok(t.map(mapper)),
```

Comment:
We want to return `Ok(None)` if `!read_through`, correct? Can you add a comment to clarify?

Reply:
Can you review the code now that I changed the name of the variants? Maybe it's more obvious now and doesn't need a comment